phoenix-core/src/main/java/org/apache/phoenix/mapreduce/util/PhoenixConfigurationUtil.java

```diff
@@ -191,6 +190,12 @@
     // provide an absolute path to inject your multi input mapper logic
     public static final String MAPREDUCE_MULTI_INPUT_MAPPER_TRACKER_CLAZZ = "phoenix.mapreduce.multi.mapper.tracker.path";
 
+    // provide control to whether or not handle MR snapshot restore on phoenix side or handled by caller externally
```

Comment:
The naming convention for all other conf property starts with MAPREDUCE. Do you mind changing the property name to something like: "MAPREDUCE_SNAPSHOT_RESTORE_EXTERNAL" ? and prefix with DEFAULT for default value ?

Reply:
Also please change the phoenix conf string from phoenix.mr.manage.snapshot.restore.externally to "phoenix.mapreduce.snapshot.restore.externally" to be consistent with other property names.